test/example/page_object.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter/widgets.dart';
```

Comment:
The page object pattern improves readability by making it so the reader does not need to understand the nuance of the finder this is especially true with more complex finders.